lisa/tools/lsmod.py

```diff
@@ -48,3 +52,16 @@ def module_exists(
             return True
 
         return False
+
+
+class KLDStat(Lsmod):
+    @property
+    def command(self) -> str:
+        return "kldstat"
+
+    def module_statically_linked(
```

Comment:
Is it the same as `module_exists`? If so, the method should be called `module_exists`. At other places, whatever the distro is, it should always use `lsmod`, and let LISA decide which tool to use.

Reply:
No this is different. `lsmod.module_exists` looks at the dynamically linked modules

Reply:
If so, should it be implemented in kernel_config? The lsmod doesn't list static built modules.

Reply:
This information in freebsd is available from kldstat command

Reply:
Should I add kldstat tool separately and not linked to lsmod?